en/core-api/cardless-credit.md

```diff
@@ -16,48 +15,51 @@ Basic integration process of Cardless Credit will be explained below.
 </article>
 </details>
 
-#### Sandbox Environment
-All the steps below are using [Midtrans Sandbox environment](https://account.midtrans.com/), not Production, to easily test the integration process. Make sure you are switching to Sandbox mode on your Midtrans account dashboard while retrieving Server Key and Client Key. Explained in [`Getting Started - Preparation`](/en/midtrans-account/overview.md).
+## Sandbox Environment
+The steps given below uses [Midtrans *Sandbox* environment](https://account.midtrans.com/) to test the integration process. Please make sure that you use the *Server Key* and *Client Key* for the *Sandbox* environment. For more details, refer to [Retrieving API Access Keys](/en/midtrans-account/overview.md#retrieving-api-access-keys).
 
-Server Key and Client Key can be retrieved on menu `Settings` > `Access Key`.
+### Steps for integration
+To integrate with *Cardless Credit* payment method, follow the steps given below.
 
-?>**Info:**
-[How to retrieve Access key](/en/midtrans-account/overview.md#retrieving-api-access-keys)
+#### 1. Sending transaction data to Charge API
 
-### Integration Step
-1. Send transaction data to API Charge.
-2. Redirect customer to credit cardless page.
-3. Create landing page after customer complete the payment.
-4. Handle transaction notification.
```

Comment:
Any reason why is this steps overview removed? It can give good overview of how many and what are the required steps. Like a grocery list, before actually going to buy the grocery.